docs/Tutorials/Create-Privacy-enabled-network.md

```diff
@@ -17,335 +17,204 @@ to encrpyt and distribute [private transactions](../Concepts/Privacy/Privacy.md)
-
-        var bytecode = "0x6060604052341561000f57600080fd5b604051602080610149833981016040528080519060200190919050505b806000819055505b505b610104806100456000396000f30060606040526000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680632a1afcd914605157806360fe47b11460775780636d4ce63c146097575b600080fd5b3415605b57600080fd5b606160bd565b6040518082815260200191505060405180910390f35b3415608157600080fd5b6095600480803590602001909190505060c3565b005b341560a157600080fd5b60a760ce565b6040518082815260200191505060405180910390f35b60005481565b806000819055505b50565b6000805490505b905600a165627a7a72305820d5851baab720bba574474de3d09dbeaabc674a15f4dd93b974908476542c23f00029";
-
-        var simpleContract = web3.eth.contract(abi);
-        var simple = simpleContract.new(42, {from:web3.eth.accounts[0], data: bytecode, gas: 0x47b760, privateFor: ["AeggpVlVsi+rxD6h9tcq/8qL/MsjyipUnkj1nvNPgTU="]}, function(e, contract) {
-            if (e) {
-                console.log("err creating contract", e);
-            } else {
-                if (!contract.address) {
-                    console.log("Contract transaction send: TransactionHash: " + contract.transactionHash + " waiting to be mined...");
-                } else {
-                    console.log("Contract mined! Address: " + contract.address);
-                    console.log(contract);
-                }
-            }
-        });
-        ```
-
-    !!! attention "Unlock account"
-        Geth accounts are locked by default. Unlock the account first before sending the transaction:
-
-        ```bash
-        geth attach new-node-1/geth.ipc
-        ```
-
-        Once connected in geth console, run:
-
-        ```javascript
-        eth.accounts
-        ```
-
-        List of the existing accounts is displayed:
+       ]
+   },
+   "alwaysSendTo": []
+}
+```
 
-        ```json
-        ["0x23214cd88f46865207fa1d2a69971a37cdbf526a"]
-        ```
+### 6. Start Tessera 0
 
-        Unlock the account:
+In the `Tessera-0` directory, start Tessera 0.
 
-        ```javascript
-        personal.unlockAccount("0x23214cd88f46865207fa1d2a69971a37cdbf526a");
-        ```
+```bash
+java -jar <path to Tessera>/tessera.jar -configfile config.json
+```
 
-        Type in your passphrase when asked for:
+### 7. Start Tessera 1
 
-        ```text
-        Unlock account 0x23214cd88f46865207fa1d2a69971a37cdbf526a
-        Passphrase:
-        ```
+In the `Tessera-1` directory, start Tessera 1.
 
-        If operation succeeded, result is:
+```bash
+java -jar <path to Tessera>/tessera.jar -configfile config.json
```

Comment:
Ask for replacement of the placeholder `<path to Tessera>` in the cmd.

Reply:
Fixed